areader/reader.go

```diff
@@ -863,7 +884,7 @@ func readAndInstall(r io.Reader, i handlers.Installer,
 		// check checksum
 		ch := artifact.NewReaderChecksum(tar, df.Checksum)
 
-		if err = i.Install(ch, &info); err != nil {
+		if err = updateStorer.StoreUpdate(ch, info); err != nil {
```

Comment:
why the change to pass by value?

Reply:
`info` is already an interface, which is always a pointer, so the old function took a double pointer. I didn't want to preserve this in the `StoreUpdate` interface.